x/concentrated-liquidity/position.go

```diff
@@ -708,7 +708,8 @@ func (k Keeper) transferPositions(ctx sdk.Context, positionIds []uint64, sender
 		if _, err := k.collectSpreadRewards(ctx, sender, positionId); err != nil {
 			return err
 		}
-		if _, _, err := k.collectIncentives(ctx, sender, positionId); err != nil {
+
+		if _, _, _, err := k.collectIncentives(ctx, sender, positionId); err != nil {
```

Comment:
Note: properly handling forfeits for transfers will be addressed in a (very light) follow-up PR. Tracked in #7776